deps/llvm.mk

```diff
@@ -336,6 +336,7 @@ endif
 ifneq ($(LLVM_LLDB_TAR),)
 	$(JLCHECKSUM) $(LLVM_LLDB_TAR)
 endif
+	-rm -rf $(LLVM_SRC_DIR)
```

Comment:
do the queue a favor and don't merge anything until https://travis-ci.org/JuliaLang/julia/builds/148541438 is done